apps/hash-graph/lib/graph/src/shared/subgraph.rs

```diff
@@ -77,7 +77,7 @@ impl Subgraph {
         &'r mut self,
         store: &impl Read<R>,
         vertex_id: &R::VertexId,
-        time_projection: &TimeProjection,
+        time_projection: &TemporalAxes,
```

Comment:
This was missed, can you "find all" for "projection", "image", and "kernel"?

Reply:
Thanks for spotting: [Fix remaining occurrences of "projection"](https://github.com/hashintel/hash/pull/1854/commits/4a26c7410090f9c4932babfe29ee41743ef38dae)